pytorch_lightning/utilities/enums.py

```diff
@@ -256,3 +256,28 @@ def interactive_compatible_types() -> List["_StrategyType"]:
     def is_interactive_compatible(self) -> bool:
         """Returns whether self is interactive compatible."""
         return self in _StrategyType.interactive_compatible_types()
+
+
+class FaultTolerantTrainingModes(LightningEnum):
+
+    DISABLED = "0"
+    AUTOMATIC = "1"
+    MANUAL = "2"
+    AUTOMATIC_ELASTIC = "3"
+    MANUAL_ELASTIC = "4"
```

Comment:
these are not supported yet, right? They should not be included

Reply:
No, done !